js/generation-form.js

```diff
@@ -31,19 +31,26 @@ const  generateForm = (Item)=> {
     featuresList.removeChild(featuresList.firstChild);
   }
   // Заводим новые приеущества
-  Item.offer.features.forEach((feature) => {
-    const featuresListItem = document.createElement('li');
-    featuresListItem.classList.add('popup__feature');
-    featuresListItem.classList.add(`popup__feature--${feature}`);
-    featuresList.appendChild(featuresListItem);
-  });
-  // Меняем описание объекта1
+  if (Item.offer.features) {
```

Comment:
```suggestion
  if (item.offer.features) {
```
`item` должен быть с маленькой